features/step_definitions/secrets.js

```diff
@@ -79,6 +79,16 @@ module.exports = function server() {
         }).then((response) => {
             this.jobId = response.body[0].id;
             this.secondJobId = response.body[1].id;
+            this.thirdJobId = typeof response.body[2] === 'object' ? response.body[2].id : null;
+            this.lastJobId = (function getLastJobId() {
```

Comment:
I really don't think you needed to change this, but this is a one-liner that should be equivalent.

```
response.body.reverse().find(b => typeof b === 'object').id;
```